### ISSUE TITLE
build failure: no method named `normalize` found for struct `pbtime::Timestamp` in the current scope

### ISSUE THREAD

**Issue by tarcieri** [fdeantoni/prost-wkt#20]
I'm getting the following error trying to include `prost-wkt-types` as a dependency:

```
error[E0599]: no method named `normalize` found for struct `pbtime::Timestamp` in the current scope
   --> .cargo/registry/src/github.com-1ecc6299db9ec823/prost-wkt-types-0.3.2/src/pbtime.rs:51:12
    |
51  |         ts.normalize();
    |            ^^^^^^^^^ method not found in `pbtime::Timestamp`
    |
   ::: target/debug/build/prost-wkt-types-7f1604c475b47791/out/pbtime/google.protobuf.rs:172:1
    |
172 | pub struct Timestamp {
    | -------------------- method `normalize` not found for this
```

**Comment by fdeantoni** [comment]
Can you include your `Cargo.toml` and `build.rs` so I can try to replicate the problem?

**Comment by fdeantoni** [comment]
In the Cargo.toml I see that `default-features` is set to `false`. This means `std` is not used, which means the `normalize` function is not available. Unfortunately this implementation comes straight from Prost, so its not in Prost-wkt-types. Luckily, though, the `master` branch of Prost seems to have already addressed this (see commit [1674ef9](https://github.com/tokio-rs/prost/commit/1674ef925ef4ad0daa09b8144742ecd1a545d72f)). I'll see if I can update Prost to the latest version of Prost for the next release of prost-wkt.
